KMaina/Fast-Food-App
# #160298066 Page to Allow an Admin Accept or Decline Orders

#### What does this PR do?
Allow an admin to accept or decline an order
#### Description of Task to be completed?
The admin should be able to 

- Accept an order made by a user

- Decline an order made by a user

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-accept-order-all-160298066`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-accept-order-all-160298066`

- In the folder where you cloned the repo, you should open the UI folder.
- Open the admin-order-edit.html to see it in your browser for review.


#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298066
